File: uri/media_tres/media_tres.go
package main

import (
	"C"
	"fmt"
)

var peso1 float64 = 2.0
var peso2 float64 = 3.0
var peso3 float64 = 5.0

func media(nota1 float64, nota2 float64, nota3 float64) float64 {
	return ((nota1 * peso1) + (nota2 * peso2) + (nota3 * peso3)) / (peso1 + peso2 + peso3)
}

func main() {
	var nota1, nota2, nota3 float64
	fmt.Scanf("%f\n", &nota1)
	fmt.Scanf("%f\n", &nota2)
	fmt.Scanf("%f\n", &nota3)
	fmt.Printf("MEDIA = %.1f\n", media(nota1, nota2, nota3))
}

File: uri/media_tres/media_tres.js
//var input = require('fs').readFileSync('stdin.txt', 'utf8');
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

var peso1 = 2.0;
var peso2 = 3.0;
var peso3 = 5.0;

function media( nota1, nota2, nota3 ){
    return ((nota1 * peso1) + (nota2 * peso2) + (nota3 * peso3)) / (peso1 + peso2 + peso3);
}

var p1 = parseFloat( lines.shift() );
var p2 = parseFloat( lines.shift() );
var p3 = parseFloat( lines.shift() );

var med = media( p1, p2, p3 );
console.log('MEDIA = ' + med.toFixed(1));